drivers/hmis/app/models/hmis/hud/enrollment_coc.rb

```diff
@@ -6,9 +6,14 @@
 
 class Hmis::Hud::EnrollmentCoc < Hmis::Hud::Base
   include ::HmisStructure::EnrollmentCoc
-  include ::Hmis::Hud::Shared
+  include ::Hmis::Hud::Concerns::Shared
   self.table_name = :EnrollmentCoC
   self.sequence_name = "public.\"#{table_name}_id_seq\""
 
   belongs_to :enrollment, **hmis_relation(:EnrollmentID, 'Enrollment')
+  belongs_to :client, **hmis_relation(:PersonalID, 'Client')
+  belongs_to :user, **hmis_relation(:UserID, 'User')
+  belongs_to :data_source, class_name: 'GrdaWarehouse::DataSource'
+
+  include ::Hmis::Hud::Concerns::EnrollmentRelated
```

Comment:
Should these be moved up to alongside the other concerns on line 9?  I'm guessing you're including them here since they contain scopes, but I'm not sure that matters.

Reply:
It didn't work above because it needs the table name and the relations to exist. I don't really understand why

Reply:
That is odd, but ok.  Maybe we should add a comment on line 17 to indicate why it's where it is.